nondex-core/src/main/java/java/util/concurrent/PriorityBlockingQueue.java

```diff
@@ -752,6 +752,29 @@ public String toString() {
         }
     }
 
+    public String toString() {
```

Comment:
I just try to minimize the changes I made so it will be possibly safer so I don't introduce more bugs..
I agree that using an iterator should also work but I didn't test that and honestly I don't know why the author of this method didn't use the iterator.
Maybe you should ask the developers why they choose not to use the iterator here. :)